src/site/content/en/fast/content-delivery-networks/index.md

```diff
@@ -0,0 +1,284 @@
+
+The tradeoff between minimizing latency and minimizing cache hit ratio is a spectrum. No particular approach is universally better; however, depending on the nature of your site and its user base, you may find that one of these approaches delivers significantly better performance than the other.
+
+It's also worth noting that CDN performance can vary significantly depending on geography, time of day, and even current events. Although it's always a good idea to do your own research on a CDN's performance, it can be difficult to predict the exact performance you'll get from a CDN.
+
+
+### Additional features
+
+CDNs typically offer a wide variety of features in addition to their core CDN offering. Commonly offered features include: load balancing, image optimization, video streaming, edge computing, and security products.
+
+
+## How to setup and configure a CDN
+
+Ideally a CDN should be used to serve your entire site. At a high-level, the setup process for this consists of signing up with a CDN provider, then updating your CNAME DNS record to point at the CDN provider. For example, the CNAME record for `www.example.com` might point to `example.my-cdn.com`. As a result of this DNS change, traffic to your site will be routed through the CDN.
+
+If using a CDN to serve all resources is not an option, a CDN can be configured to only serve a subset of resources - for example, only static resources. This is accomplished by creating a separate CNAME record that will only be used for resources that should be served by the CDN. For example, creating a `static.example.com` CNAME record that points to `example.my-cdn.com`. The URLs of resources that should be served by the CDN should be rewritten to point at the `static.example.com` subdomain that you just created.
+
+Although your CDN will be set up at this point, there will likely be inefficiencies in your configuration. The next two sections of this article will explain how to get the most out of your CDN by increasing cache hit ratio and enabling performance features.
+
+
+## Improving cache hit ratio
+
+An effective CDN setup will serve as many resources as possible from the cache. This is commonly measured by cache hit ratio (CHR). Cache hit ratio is defined as the number of cache hits divided by the number of total requests during a given time interval.
+
+A freshly initialized cache will have a CHR of 0 but this increases as the cache is populated with resources. A CHR of 90% is a good goal for most sites. 
+
+When optimizing CHR, the first thing to verify is that all cacheable resources are being cached and cached for the correct length of time. This is a simple assessment that should be undertaken by all sites.
```

Comment:
What makes it simple? Is it because all good CDNs provide this feature? If that's the case, I think we should mention something along those lines. In other words we should make it clear that it's a red flag if a supposed CDN doesn't provide CHR data.